model/clusters_mgmt/v1/node_pool_type.model

```diff
@@ -38,4 +41,7 @@ class NodePool {
 
     // The subnet upon which the nodes are created.
     Subnet String
+
+    // Adds additional information about the NodePool status when Replicas != CurrentReplicas.
+    Message String
```

Comment:
Perhaps this should be a struct?
Also it sounds very generic. Can you provide an example of what this msg will be?

Reply:
The message is open.
NodePools don't have a lifecycle; the status is the CurrentReplicas; this message is a custom message that would explain the situation.

It's generic. Then a string is the best option for this case.

Reply:
But +1.
It makes sense if status grows.
Thanks